### PR TITLE
Add security magboots

### PR DESCRIPTION
**Pull request by Radezolid**
## About the PR
Added the security magboots. The sprites were done by [ProphetCrow](https://github.com/ProphetCrow) and used here with their permission, this PR is a remake of their abandoned PR https://github.com/DeltaV-Station/Delta-v/pull/1490.

## Why / Balance
I've seen lube being used by nuclear operatives a lot lately and it leaves almost no counterplay from the crew against the nukies.
These aren't meant for security to wear all the time and should be considered powergaming, since it's the same reason they shouldn't wear EVA suits all the time.

## Technical details
Modified the `MagnetsTech` tech to make the security magboots researcheable, added recipe for the sec techfab to be able to print it them, modified the table for the HOS hardsuit spawn, and added the magboot to the sec suit storages, including warden's locker.

## Media

https://github.com/user-attachments/assets/5a2de7e6-8406-473e-92b3-cee7781822cd

Security TechFab:
image

Research:
image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

Hopefully none.

**Changelog**

:cl: Crow, Radezolid
- add: Added security magboots, they should only be used for EVA or situations that warrant no slips.

### CODE REVIEW

File: Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from Engineering Magboots from tgstation at commit https://github.com/tgstation/tgstation/commit/7e4e9d432d88981fb9bb463970c5b98ce85c0abe , edited by ProphetCrow (Github)",
+
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json]: delete

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json (hunk 1 above)
Comment (Radezolid) [reply Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json]: That's the licensing from the spriter, gonna ask them what they want, dunno.

Comment (Radezolid) [reply Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json]: The license is fine, why do you want it removed?

Comment (deltanedas) [reply Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json]: delete the empty line, not the license

Comment (Radezolid) [reply Resources/Textures/DeltaV/Clothing/Shoes/Boots/magboots-security.rsi/meta.json]: Pfff

File: Resources/Prototypes/Catalog/Fills/Lockers/security.yml [hunk 2]
```diff
@@ -32,6 +32,7 @@
         amount: 2
       - id: RemoteSignaller
         amount: 2
+      - id: ClothingShoesBootsSecurityMagboots # DeltaV - Added security magboots.
```
Comment (deltanedas) [inline Resources/Prototypes/Catalog/Fills/Lockers/security.yml:35]: should be in the suit storage instead, and needs updating to have 2 of them now

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Review by deltanedas — APPROVED** [review]

**Pull request merged**